src/coverlet.collector/DataCollection/CoverletCoverageCollector.cs

```diff
@@ -130,12 +130,11 @@ private void OnSessionEnd(object sender, SessionEndEventArgs e)
                 _eqtTrace.Verbose("{0}: SessionEnd received", CoverletConstants.DataCollectorName);
 
                 // Get coverage reports
-                string coverageReport = _coverageManager?.GetCoverageReport();
+                var coverageReports = _coverageManager?.GetCoverageReports();
 
                 // Send result attachments to test platform.
-                var attachmentManager = new AttachmentManager(_dataSink, _dataCollectionContext, _logger, _eqtTrace, this.GetReportFileName());
-                attachmentManager?.SendCoverageReport(coverageReport);
-
+                var attachmentManager = new AttachmentManager(_dataSink, _dataCollectionContext, _logger, _eqtTrace);
+                coverageReports?.ToList().ForEach(x => attachmentManager?.SendCoverageReport(x.Item1, x.Item2));
```

Comment:
`x` => `report` and use deconstructed name `report`/`fileName`

Reply:
Done